http/cancel.go

```diff
@@ -0,0 +1,31 @@
+package http
```

Comment:
I would call this package api or dune

because think about it from the users perspective, they are calling `dune.WaitForCompletion` or `api.QueryExecute` 

also, you can have a "QueryExecution" being a struct/interface with "Cancel()" , "Status()", or "Results()" methods